sb_vision/native/apriltag/detector.py

```diff
@@ -1,14 +1,19 @@
 """AprilTag detector wrapper."""
 
+from typing import Any, Iterator, Tuple
+
 from PIL import Image
 
 from ._apriltag import ffi, lib
 
+if False:
```

Comment:
er....

Reply:
This is the way to specify typing-only imports.

Reply:
Python 3.6 has a better solution (a constant `TYPING` which type-checkers can treat as `True` but which the runtime treats as `False`), but we need Python 3.5 support.

Reply:
It turns out that the constant I was thinking of ([`TYPE_CHECKING`](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING)) was actually introduced in Python 3.5.2, so we can _probably_ use it. I'll check which patch version of Python is on the Pis before switching over though.

Reply:
It turns out that the Pis are running python 3.5.3, so this should be fine to use.

Reply:
159b74c